### ISSUE TITLE
Feature request: Hls support

### ISSUE THREAD

**Issue by benwoodward** [meigo/svelte-video-player#6]
This lib looks great. Would be great if there was Hls support—is this something you're planning on adding? ..I'm trying to figure out what would be involved in adding this functionality..

**Comment by meigo** [comment]
Yes, thanks for pointing this out, but not before this hot summer is over 🌞

**Comment by marksteven** [comment]
I too would like HLS support :)
